tools/packaging/debian/compat

```diff
@@ -0,0 +1 @@
+11
```

Comment:
do you really need the emptychangelog file?

Reply:
I will try to make it would without it

Reply:
removed